Meadowlark-js/backends/meadowlark-postgres-backend/src/repository/Read.ts

```diff
@@ -0,0 +1,43 @@
+// // SPDX-License-Identifier: Apache-2.0
+// // Licensed to the Ed-Fi Alliance under one or more agreements.
+// // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// // See the LICENSE and NOTICES files in the project root for more information.
+
+import { Client, Result } from 'pg';
+import { GetResult, GetRequest, DocumentIdentity } from '@edfi/meadowlark-core';
+
+// @ts-ignore
```

Comment:
What doesn't TS like about this?

Reply:
in it's current state, it's fine and I forgot to remove the ignore, not remembering what caused me to put that there in the first place